sql/mob_droplist.sql

```diff
@@ -26775,6 +26775,13 @@ INSERT INTO `mob_droplist` VALUES (3284,0,0,1000,19120,@UNCOMMON); -- Avis (Unco
 -- ZoneID:  88 - Olgoi-Khorkhoi
 INSERT INTO `mob_droplist` VALUES (3285,0,0,1000,18768,@COMMON); -- Poppet Katars (Uncommon, 15%)
 
+-- ZoneID: 81, 82, 84, 89, 91, 95, 96 - Dark Ixion
```

Comment:
I don't know if this is acceptable, or if this messes with the auto-generated formatting of this file? @claywar ?

Reply:
Xaver informed me that I should just leave no comment, but i've since moved the sql [to a smaller PR](https://github.com/LandSandBoat/server/pull/4706)